src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --header-size: 60px;
+}
+
 body {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+}
+
+.header__container {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__logo-link {
+  display: block;
+  height: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 0;
   margin: 0;
+  display: block;
+  font-weight: 500;
+  height: 60px;
+  font-size: 12px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You should add a fallback font in case 'Roboto' doesn't load. This is a good practice to ensure that your website has a similar look on different devices even if the main font fails to load. For example, you can modify this line to `font-family: Roboto, Arial, sans-serif;`.